app/public/js/localSprites.js

```diff
@@ -173,21 +201,39 @@ function updateLocalSprite(localSprite) {
             if (!(moveSpriteInDirection(localSprite, localSprite.vy, TILE_DOWN))) localSprite.shouldBeRemoved = true;
         }
     }
-    //update trigger zones
-    if (localSprite.type === 'SPRITE_TYPE_TRIGGER') {
-        if (rectanglesOverlap(localSprite.hitBox, getGlobalSpriteHitBox(localSprite.target)) && localSprite.cooldownTimer === 0) { //when I changed "localSprite.hitBox" to "getGlobalSpriteHitBox(localSprite), this stopped working.
-            if (localSprite.spawnedObjectType === 1) addHomingFireballSprite(localSprite.spawnedObjectXOffset, localSprite.spawnedObjectYOffset, localSprite.target);
-                if (localSprite.xForce) localSprite.target.vx += localSprite.xForce;
-                if (localSprite.yForce) localSprite.target.vy += localSprite.yForce;
-                localSprite.cooldownTimer++;
+    //update triggers
+    if (localSprite.type === TRIGGER_TYPE_FORCE) {
+        if (rectanglesOverlap(localSprite.hitBox, getGlobalSpriteHitBox(localSprite.target)) && localSprites.length <2/*canTriggerTrigger(localSprite)*/) {
+            if (localSprite.forceType === FORCE_AMP) {
+                if (localSprite.target.vx) localSprite.target.vx *= localSprite.xForce;
+                if (localSprite.target.vy) localSprite.target.vy *= localSprite.yForce;              
+            }
+            if (localSprite.forceType === FORCE_FIXED) {
+                if (localSprite.target.vx) localSprite.target.vx += localSprite.xForce;
+                if (localSprite.target.vy) localSprite.target.vy += localSprite.yForce;
+            }
+            localSprite.notReadyToTriggerUntil = now() + localSprite.cooldownInMS;
```

Comment:
If you checked `if (localSprite.cooldownInMS)` before this then this would support triggers without cooldowns. I can imagine wanting a force field with no cooldown.